showcase/app/templates/components/app-header/index.hbs

```diff
@@ -337,37 +408,60 @@
 
   <Shw::Text::H2>Base elements</Shw::Text::H2>
 
-  <Shw::Text::H3>SideNav::Header::HomeLink</Shw::Text::H3>
+  <Shw::Text::H3>AppHeader::HomeLink</Shw::Text::H3>
 
   <Shw::Text::H4>Content</Shw::Text::H4>
 
   <Shw::Flex as |SF|>
     <SF.Item @label="Icon">
       <div class="hds-app-header">
-        <Hds::AppHeader::HomeLink @icon="hashicorp" @ariaLabel="HashiCorp" @href="#" />
+        <Hds::AppHeader::HomeLink @icon="hashicorp" @text="HashiCorp" @isIconOnly={{true}} @href="#" />
       </div>
     </SF.Item>
 
     <SF.Item @label="Custom color">
       <div class="hds-app-header">
         <Hds::AppHeader::HomeLink
           @icon="boundary"
-          @ariaLabel="Boundary"
+          @text="Boundary"
+          @isIconOnly={{true}}
           @color="var(--token-color-boundary-brand)"
           @href="#"
         />
       </div>
     </SF.Item>
   </Shw::Flex>
 
+  <Shw::Text::H4>Text</Shw::Text::H4>
+
+  <Shw::Flex as |SF|>
+    <SF.Item>
+      <div class="hds-app-header">
+        <Hds::AppHeader::HomeLink @icon="hashicorp" @text="HashiCorp" @href="#" @title="Admin UI" />
+      </div>
+    </SF.Item>
+
+    <SF.Item>
+      <div class="hds-app-header">
+        <Hds::AppHeader::HomeLink @icon="terraform" @text="HashiCorp" @href="#" @title="Terraform Admin Console" />
+      </div>
+    </SF.Item>
+  </Shw::Flex>
+
   <Shw::Text::H4>States</Shw::Text::H4>
 
   <Shw::Flex as |SF|>
     {{#let (array "default" "hover" "active" "focus" "disabled") as |states|}}
       {{#each states as |state|}}
         <SF.Item @label={{state}}>
           <div class="hds-app-header">
-            <Hds::AppHeader::HomeLink @icon="hashicorp" @ariaLabel="HashiCorp" @href="#" mock-state-value={{state}} />
+            <Hds::AppHeader::HomeLink
```

Comment:
This disabled state variant is throwing a color contrast error in the automated a11y tests, which wasn't throwing an error previously because of the absence of text. I have a couple of questions:
- We don't have this in Figma, is there a reason why it's needed here?
- If it is needed for testing purposes, is there a way to ignore the automated color contrast test for disabled elements?